internal/graphql/resolver/schema.resolvers.go

```diff
@@ -46,28 +46,28 @@ func (r *queryResolver) GetOperatorStatus(ctx context.Context) (*model.OperatorS
 	return r.getOperatorStatus(ctx)
 }
 
-func (r *subscriptionResolver) ListenToAddonEvents(ctx context.Context, selector *model.MeshType) (<-chan []*model.AddonList, error) {
+func (r *subscriptionResolver) ListenToAddonState(ctx context.Context, selector *model.MeshType) (<-chan []*model.AddonList, error) {
 	if selector != nil {
-		return r.listenToAddonEvents(ctx)
+		return r.listenToAddonState(ctx)
 	}
 
 	return nil, ErrInvalidRequest
 }
 
-func (r *subscriptionResolver) ListenToControlPlaneEvents(ctx context.Context, filter *model.ControlPlaneFilter) (<-chan []*model.ControlPlane, error) {
+func (r *subscriptionResolver) ListenToControlPlaneState(ctx context.Context, filter *model.ControlPlaneFilter) (<-chan []*model.ControlPlane, error) {
```

Comment:
Is there a meaningful difference between control plane state and add-on state? Why are the two tracked independently?

Reply:
They are on different pages, thus different subscriptions (websocket. connections). so had to segregate them.